### PR TITLE
Update working-with-3-d-models.md

### PR DESCRIPTION
**Pull request by walbourn**
Note that this feature was removed without warning in 17.9.3

### CODE REVIEW

**Comment by Jak-MS** [comment]
@Mikejo5000 
- Can you review this PR?
- IMPORTANT: When this content is ready to merge, you must add `#sign-off` in a comment or the approval may get overlooked.


#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Review by Mikejo5000 — APPROVED** [review]
Thanks for the correction!

**Comment by Mikejo5000** [comment]
#sign-off

File: docs/designers/working-with-3-d-models.md [hunk 1]
```diff
@@ -12,6 +12,8 @@ ms.subservice: ui-designers
 
 You can use the Model Editor in Visual Studio to create 3D models. You can use the models in your DirectX-based game or app.
 
+> Visual Studio 2022 17.9.3 removed support for the Model Editor and MeshContentTask in due to security concerns in the Autodesk FBX SDK. See [CVE-2023-27911](https://msrc.microsoft.com/update-guide/vulnerability/CVE-2023-27911).
```
Comment (Court72) [inline docs/designers/working-with-3-d-models.md:15]:
```suggestion
> [!NOTE]
> Visual Studio 2022 17.9.3 removed support for the Model Editor and MeshContentTask due to security concerns in the Autodesk FBX SDK. See [CVE-2023-27911](https://msrc.microsoft.com/update-guide/vulnerability/CVE-2023-27911).
```

**Pull request merged**